chats/models.py

```diff
@@ -17,4 +17,24 @@ def __str__(self):
         return f'{self.author.username} : {self.body}'
     
     class Meta:
-        ordering = ['-created_at']
\ No newline at end of file
+        ordering = ['-created_at']
+
+class PrivateChatRoom(models.Model):
+    room_name = models.CharField(max_length=128, unique = True)
+
+    def __str__(self):
+        return self.group_name        
```

Comment:
```suggestion
        return self.group_name        
```
這裡不是 return `room_name` 嗎？

Reply:
已修正為 room_name.